controllers/gce/controller/controller_test.go

```diff
@@ -53,7 +53,8 @@ func defaultBackendName(clusterName string) string {
 // newLoadBalancerController create a loadbalancer controller.
 func newLoadBalancerController(t *testing.T, cm *fakeClusterManager) *LoadBalancerController {
 	kubeClient := fake.NewSimpleClientset()
-	ctx := NewControllerContext(kubeClient, api_v1.NamespaceAll, 1*time.Second)
+	eventRecorder := utils.NewEventRecorder(kubeClient)
+	ctx := NewControllerContext(kubeClient, api_v1.NamespaceAll, 1*time.Second, eventRecorder)
```

Comment:
For my knowledge, why the change to pass eventRecorder in context rather than in NewController? Why is kubeclient passed in both?